tox.ini

```diff
@@ -23,22 +21,24 @@ install_command = python -m pip install --no-cache-dir {opts} {packages}
 commands_pre =
     rm -Rf tests/_reports/coverage*
     atlas: pip install -i https://test.pypi.org/simple/ atlas4py
-commands =
-    pytest --cache-clear --cov -n auto -v -- {posargs}
-    pytest --doctest-modules --cov --cov-append -n auto -v src/
 commands_post =
-    coverage report --rcfile=setup.cfg
-    coverage html --rcfile=setup.cfg --show-contexts
```

Comment:
Why removing the `html` reports here? They are very useful when running `tox` locally during development

Reply:
I don't use it locally myself, but I restored it for everyone who does.